### PR TITLE
basic django server that serves basic home page with dockerfile

### PR DESCRIPTION
**Pull request by ZachSellman**
Man these dockerfiles are messing me up... never seem to find good examples. Everything's working here now finally. Just needs to be run with "docker run -p 8000:8000 <image> " to server running in the container externally.

I figured I'd wait to see if you have significant notes to this pull before digging too far into the next steps.

### CODE REVIEW

File: core/core/settings.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    },
+]
+
+WSGI_APPLICATION = "core.wsgi.application"
+
+
+# Database
+# https://docs.djangoproject.com/en/4.2/ref/settings/#databases
+
+DATABASES = {
```
Comment (justincely) [inline core/core/settings.py:77]: fine for now - will be good to setup a postgres engine at some point too.  Wont use sqlite3 in production.

File: core/db.sqlite3
Comment (justincely) [inline core/db.sqlite3:1]: don't commit the DB.

**Review by justincely — CHANGES_REQUESTED** [review]
Mostly commits to remove, and you'll need to modify the `staticcheck` CI step since this isn't a go project anymore.  I'd recommend replacing it with something like `flake8` or any of the other tools in the ecosystem.

**Comment by justincely** [comment]
the `core/core` project path is a little weird too, presumably an artifact of the quick start or something.  but can always pull it out later if need be.

File: core/core/__pycache__/__init__.cpython-310.pyc
[the remark this answers is not in the document]
Comment (ZachSellman) [reply core/core/__pycache__/__init__.cpython-310.pyc:1]: Yup, forgot to bring in the boilerplate .gitingore file

**Comment by ZachSellman** [comment]
Yeah most of this is just the default project files django creates. The core/core path is supposedly standard (based on the courses I've taken at least) where you just provide the name of the project and the django-admin startproject command creates that structure.

**Review by justincely — APPROVED** [review]

**Pull request merged**